File: backend/chat-service/src/main/java/it/roadies/chat_service/data/dto/response/ConversationResponseDTO.java
package it.roadies.chat_service.data.dto.response;

import lombok.*;

import java.time.LocalDateTime;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ConversationResponseDTO {

    private UUID id;
    private String travelerId;
    private String organizerId;
    private UUID travelId;
    private LocalDateTime createdAt;
}

File: android-app/app/src/main/java/it/roadies/android_app/client/models/chat/ConversationResponseDTO.kt
package it.roadies.android_app.client.models.chat

import com.google.gson.annotations.SerializedName

data class ConversationResponseDTO(
    @SerializedName("id")
    val id: String,

    @SerializedName("travelerId")
    val travelerId: String,

    @SerializedName("organizerId")
    val organizerId: String,

    @SerializedName("createdAt")
    val createdAt: String? = null
)